api/Request.json

```
@@ -219,10 +219,64 @@
             }
           }
         },
+        "init_duplex_parameter": {
+          "__compat": {
+            "description": "`init.duplex` parameter",
+            "spec_url": "https://fetch.spec.whatwg.org/#dom-requestinit-duplex",
+            "support": {
+              "chrome": {
+                "version_added": "105"
+              },
+              "chrome_android": {
+                "version_added": false
+              },
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
+                "impl_url": "https://bugzil.la/1792434"
+              },
+              "firefox_android": "mirror",
+              "ie": {
+                "version_added": false
+              },
+              "nodejs": [
+                {
+                  "version_added": "19.1.0"
+                },
+                {
+                  "version_added": "18.13.0",
+                  "version_removed": "19.0.0"
+                },
+                {
+                  "version_added": "16.19.0",
+                  "version_removed": "17.0.0"
+                }
```

I guess we can ignore <18.0.0, because the fetch API was behind a flag?

```suggestion
                }
```

---

This will also resolve this problem raised by the linter:

```
Consistency - 1 problem (1 error, 0 warnings):
 ✖ api.Request.Request - Error → Basic support in nodejs was declared implemented in a later version (18.0.0) than the following sub-feature(s):
        → api.Request.Request.init_duplex_parameter: 16.19.0
```

---

yes, mu test in v16.20.0 confirmed that it is not supported in v16